docs/reference/configuration.txt

```diff
@@ -168,7 +168,10 @@ can be configured.
           # on 2.4 and 2.6 is :plain)
           auth_mech: :scram
 
-          # The database or source to authenticate the user against. (default: admin)
+          # The database or source to authenticate the user against.
+          # If no auth source is specified, then a default will be set by the client
```

Comment:
I am going to change "client" to "driver" because I think this is helpful to clarify that the source is determined by a whole other project from Mongoid.

Reply:
I agree 👍

Reply:
Just changed it now